### DISCUSSION TITLE
New E1 Zoom Firmware has wrong extension/name

### DISCUSSION THREAD

**Discussion by WoosterInitiative** [AT0myks/reolink-fw-archive#29] [Firmware issues]
I have an E1 Zoom (hardware no. IPC_515BSD6) and saw that there's new firmware here (and on Reolink's website). However, previous firmwares have a file extension of *ipc_515bsd6* while this new one uses *pak* like other cameras (I upgraded my RLC-510a's earlier today). It won't even upload the wrongly named file.

My question is this: can I just rename the file extension? Is there any real harm in doing that?

If that is true, a note should be added for the new firmware re: this needed step. A PDF is included in the download link from Reolink, but it does not mention this.

Update 1: Alright, so, that seemed to work! I'll create an issue to have the docs fixed. I'm not sure the best way to do it or I'd do my own PR.

**Comment by AT0myks** [comment]
It's not uncommon to see a firmware with the hardware version as the extension, but I think that's the first time I read that the extension being `.pak` prevents an update. I wouldn't be surprised if that's the same for cameras other than the E1 Zoom, so maybe we should add something that applies to all of them just in case. Did you try other extensions before the update worked?

**Comment by emasty** [comment]
I figured that renaming the .pak file to .IPC_515BSD6 only works if you have a certain firmware already installed.

I first tried to install the upgrade renaming the file from firmware version v3.0.0.136_20121108. But that didn't work. The updater responded that the update cannot be done. 

Installing the updates available in this repository solved the issue for me. I did upgrade step by step, but I think that version v3.0.0.1107_22070508 is the important one. After upgrading to that version, renaming the latest .pak file and providing it to the upgrader worked.

Many thanks for this archive! I wouldn't have been able to upgrade the camera without the files found in this repository...

**Reply by emasty** [reply]
BE CAREFUL!
The update did disable the web-ui!

To enable the webui again, you have to use the windows or macos client and then enable the port again at Settings > Network > Advanced > Server Settings.

I know that is not a Reolink support forum, but maybe it helps someone...

**Reply by nh2** [reply]
I also found you have to rename the file to e.g. `IPC_515BSD6.2356_23062008.E1-Zoom.OV05A10.5MP.WIFI1021.PTZ.REOLINK.IPC_515BSD6`. You cannot simply rename it to `upgrade.IPC_515BSD6`. The upgrader will fail with `Upgrade Failed. Failed to recognize the file format` on that, even when you're already on `v3.0.0.1107_22070508`.

**Reply by nh2** [reply]
> The update did disable the web-ui!

For me the upgrade did NOT disable the HTTPS access.

I made sure to not click the checkbox that resets the config though.

(I upgraded `v3.0.0.247_21040708` ->  `v3.0.0.716_21112408` -> `v3.0.0.1107_22070508` -> `v3.0.0.2356_23062008`.)

What it did though is that the `v3.0.0.1107_22070508` -> `v3.0.0.2356_23062008` upgrade made HTTP redirect to HTTPS (and thus presented a HTTPS trust error, which one could ignore to access the web GUI).
